rfcs/text/0005-host-metric-fields.md

```diff
@@ -74,6 +74,160 @@ Stage 1: Provide a high-level description of example sources of data. This does
 Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting.
 -->
 
+Example source document from AWS EC2 instance:
```

Comment:
Quick note on the format of the RFC. I think it would be appropriate to add a few of these examples to the RFC, but as different files.

If you create directory `rfcs/text/0005` to match the RFC number, you're welcome to drop an example JSON file for each of the sources you want to discuss.

This will keep the RFC document more readable, while still providing all of the relevant details.

You can then reference these files from the document via relative links. E.g. see how we link to the "session.yml" field definitions in [the session RFC here](https://github.com/elastic/ecs/pull/935/files#diff-a062b24940a067b1921bc6b79bc054215392cce1d3dee1b2caaf804e587b6247R25)

Reply:
Done, thank you!